docling/backend/genos_msword_backend.py

```
@@ -524,52 +557,104 @@ def _resolve_part_by_rid(self, owner_part, rId):
+        partname 이 달라도 실제 콘텐츠가 같으면 동일 시그니처가 나오도록 하여
+        섹션·페이지마다 반복되는 동일 머리말/꼬리말을 1회만 출력하기 위함.
+        내용이 없으면 빈 문자열을 반환한다.
+        """
+        ns = {
+            "w": "http://schemas.openxmlformats.org/wordprocessingml/2006/main",
+            "a": "http://schemas.openxmlformats.org/drawingml/2006/main",
+            "r": "http://schemas.openxmlformats.org/officeDocument/2006/relationships",
+        }
+        try:
+            texts = [t.text for t in root_el.findall(".//w:t", namespaces=ns) if t.text]
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)